robomaker_simulation_msgs/README.md

```diff
@@ -7,12 +7,52 @@ inside of an AWS RoboMaker simulation.
 
 You can tag, untag, and list tags in your simulation job from the ROS command-line or in your ROS application while it is running. See [AWS RoboMaker documentation](https://docs.aws.amazon.com/robomaker/latest/dg/simulation-job-tags.html). You must have an IAM role with the permissions below. Replace account# with your account number.
 
-## SetTag
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "robomaker:TagResource",
+                "robomaker:UntagResource",
+                "robomaker:ListTagsForResource"
+            ],
+            "Resource": [
+                "arn:aws:robomaker:*:account#:simulation-job/*"
```

Comment:
I like to use `< >` around variables in strings like this since `< >` are not valid characters normally.

Can you update `account#` to `<account #>`